lib/streamlit/metrics_util.py

```diff
@@ -0,0 +1,76 @@
+import os
+import platform
+import subprocess
+import threading
+import uuid
+
+from typing import Optional
+
+from streamlit import file_util
+
+def _get_machine_id_v1():
+    """Get the old machine id"""
```

Comment:
Looks good - but can you add some docs explaining what this means? (Like, what does a machine ID do for us, why did we have an old one, why do we have a new one?)